modules/Messenger/messageWall_view.php

```diff
@@ -28,16 +28,9 @@
     $dateFormat = $_SESSION[$guid]['i18n']['dateFormatPHP'];
     $date = isset($_REQUEST['date'])? $_REQUEST['date'] : date($dateFormat);
 
-    $extra = '';
-    if ($date == date($dateFormat)) {
-        $extra = __($guid, "Today's Messages").' ('.$date.')';
-    } else {
-        $extra = __($guid, 'View Messages').' ('.$date.')';
-	}
-
-    echo "<div class='trail'>";
-    echo "<div class='trailHead'><a href='".$_SESSION[$guid]['absoluteURL']."'>".__($guid, 'Home')."</a> > <a href='".$_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/'.getModuleName($_GET['q']).'/'.getModuleEntry($_GET['q'], $connection2, $guid)."'>".__($guid, getModuleName($_GET['q']))."</a> > </div><div class='trailEnd'>$extra</div>";
-	echo '</div>';
+    $page->breadcrumbs->add(($date === date($dateFormat)) ?
+        __('Today\'s Messages').' ('.$date.')' :
```

Comment:
Hmm, I wonder if switching this to a string-literal and adding a slash will cause gettext to treat it as a new string 🤔 (I prefer string-literals too!) Perhaps in this case it may be best to keep it in the original format, to avoid accidentally generating a new translation string.

```suggestion
        __("Today's Messages").' ('.$date.')' :
```

Reply:
If you're using `xgettext` to extract string, it recognizes escaped quote within a string literal.

I tried with a `test.php` with these lines:
```php
<?php

gettext('hello world');
gettext('hello elsa\'s world');
gettext("hello elsa's world");
```

The I run `xgettext --from-code=UTF-8 -o test.pot test.php`. And this is the result:
```pot
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-11-08 13:55+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.php:4
msgid "hello world"
msgstr ""

#: test.php:5 test.php:6
msgid "hello elsa's world"
msgstr ""
```

Reply:
Nice! Learnin' new stuff every day 😃 thanks for checking that out!